src/components/Facepile/Facepile.Props.ts

```diff
@@ -8,6 +8,18 @@ export interface IFacepileProps extends React.Props<Facepile> {
    * is fixed at PersonaSize.extraSmall regardless of what is specified.
    */
   personas: IFacepilePersona[];
+
+  /** Maximum number of personas to show */
+  maxDisplayablePersonas?: number;
+
+  /** Show add person button */
+  showAddButton?: boolean;
```

Comment:
One suggestion here could be:

instead have addButtonProps: IButtonProps

if provided, the button will be displayed and the given props will be mixed into it.

That allows the caller to provide more than just onClick, like data-automation-id.